apiserver/metricsender/metricsender.go

```diff
@@ -23,8 +23,9 @@ type MetricSender interface {
 }
 
 var (
-	defaultMaxBatchesPerSend              = 10
-	defaultSender            MetricSender = &HttpSender{}
+	defaultMaxSendsPerInvocation              = 5
```

Comment:
I reckon this should be much higher, just in case we have a backlog we're trying to clear, like 100?